app/db/entities/group-permission.ts

```diff
@@ -24,15 +24,15 @@ export class GroupPermission {
   groupID: number;
 
   @ValidateNested()
-  @ManyToOne(type => Group, group => group.id, { eager: true })
+  @ManyToOne(type => Group, group => group.id, { eager: true, onDelete: "CASCADE" } )
```

Comment:
Confirming, this cascade will delete the relation in the pivot table but not delete the group or something crazy?